doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1632,7 +1632,8 @@ name of the variable (here :g:`n`) is chosen based on :g:`T`.
 
    This variant allows to refer to an existential variable which was not named
    by the user. The :n:`@natural` argument is the position of the existential variable
-   from right to left in the goal. Because this variant is not robust to slight
+   from right to left in the conclusion of the goal (use one of the variants below to select
+   an existential variable in a hypothesis). Because this variant is not robust to slight
```

Comment:
```suggestion
   from right to left in the conclusion of the goal.  (Use one of the variants below to select
   an existential variable in a hypothesis.) Because this variant is not robust to slight
```